src/cloud-api-adaptor/podvm-mkosi/Dockerfile.podvm_docker_provider

```diff
@@ -1,12 +1,21 @@
 # Adapted from https://github.com/kubernetes-sigs/kind/blob/main/images/base/Dockerfile
 
 ARG BASE_IMAGE=registry.fedoraproject.org/fedora:40
+
+FROM $BASE_IMAGE AS iptables
+
+RUN echo "Building iptables-wrapper... " \
+    && dnf install -y git golang \
+    && git clone https://github.com/kubernetes-sigs/iptables-wrappers.git /iptables \
+    && cd /iptables \
+    && make BIN_DIR=. build
+
```

Comment:
I'm wondering why some of us didn't face the issue. Is it because we were using Fedora or Ubuntu ?

Reply:
I use RHEL 9.5 for my worker node. The iptables-nft package is installed on it. The current `podvm-docker-image` contains the iptables-legacy package only and no kernel module packages.

In my environment, necessary kernel modules for iptables-legacy are not loaded.
When I manually load the necessary kernel modules as follows, the `podvm-docker-image` 
works file without the error I reported.

```
sudo modprobe ip_tables
sudo modprobe iptable_filter
sudo modprobe iptable_raw
```

So, my guess is that iptables-legacy is used on the environments where the error is not observed.

Reply:
Thanks @yoheiueda. That explains